src/main/groovy/org/liquibase/groovy/delegate/PreconditionDelegate.groovy

```diff
@@ -161,6 +161,9 @@ class PreconditionDelegate {
             } else if ( key == "onError" ) {
                 preconditions.onError = ErrorOption."${paramValue}"
             } else if ( key == "onUpdateSQL" ) {
+                println "Warning: ChangeSet '${changeSetId}': onUpdateSQL parameter has been deprecated, and may be removed in a future release. Please use onUpdateSql instead of onSqlOuptut."
```

Comment:
@stevesaliman do you want deprecated `onUpdateSQL` property in groovy-dsl? or allow `onUpdateSQL` properties to be set without warning-logging?

Reply:
Version 4.0.0 of the plugin removes support of all versions of liquibase prior to version 4.24, so I don't think we need to keep the deprecated parameter. 

We do need to keep the newer `onUpdateSql` parameter, however.  It appears to be the same as `onSqlOutput`, just for different changelog types.  I'll take care of this fix.

Reply:
ok, thanks!